tests/Oryx.Integration.Tests/Node/NodeNextJsAppTest.cs

```diff
@@ -93,7 +93,7 @@ public async Task CanBuildAndRun_BlogStarterNextJsApp_UsingZippedNodeModules()
                 .AddCommand(
                 $"oryx build {appDir} -i /tmp/int -o /tmp/out --platform nodejs " +
                 $"--platform-version {nodeVersion} -p compress_node_modules={compressFormat}")
-                .AddCommand($"cp -rf /tmp/out/* {appOutputDir}")
+                .AddCommand($"cp -rf /tmp/out/. {appOutputDir}")
```

Comment:
The `.` here is to enable copying hidden files and folders too. It is relevant here since there is a `.next` folder